pkg/curatedpackages/curatedpackages.go

```diff
@@ -0,0 +1,61 @@
+package curatedpackages
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/go-logr/logr"
+
+	"github.com/aws/eks-anywhere-packages/pkg/artifacts"
+	"github.com/aws/eks-anywhere-packages/pkg/bundle"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/helm"
+	"github.com/aws/eks-anywhere/pkg/version"
+)
+
+func NewRegistry(ctx context.Context, deps *dependencies.Dependencies, registry, kubeVersion string) (BundleRegistry, error) {
```

Comment:
Maybe this logic would be better suited for the dependency factory

Reply:
I thought the factory was used for bootstrapping any type of dependency required from the release manifest?